.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
     - pwd
     - diff -u <(echo -n) <(gofmt -d ./)
     - go test -v ./...
+    - chmod u+x ./go-get-tests.sh
```

Comment:
Why not just check it into the repo as +x?

Reply:
It is right now, but you never know if a windows commit comes in that backs it out. Just being a bit defensive.